src/caffe/layers/prelu_layer.cu

```diff
@@ -83,13 +88,13 @@ void PReLULayer<Dtype>::Backward_gpu(const vector<Blob<Dtype>*>& top,
     Dtype* slope_diff = this->blobs_[0]->mutable_gpu_diff();
     int cdim = channels * dim;
     Dtype dsum = 0.;
-    for (int n = 0; n < bottom[0]->num(); ++n) {
+
```

Comment:
After removing the loop, let's delete the fix the indent here. No block indent needed here after removing the loop.